### DISCUSSION TITLE
[Feature] Import media from gphotos-sync backup

### DISCUSSION THREAD

**Discussion by Feriman22** [immich-app/immich#11978] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Hi,

It would be a great feature if I could import the backup by [gphotos-sync](https://github.com/gilesknap/gphotos-sync) into immich.

Is it possible with the current version?

Thanks!
Feriman

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
The gphotos API gives you non-original versions of your files, with a worse quality and some metadata removed. We recommend using https://takeout.google.com/ with https://github.com/simulot/immich-go.
